travel-event-host-app/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,73 @@
+import NextAuth from 'next-auth/next';
+import CredentialsProvider from 'next-auth/providers/credentials';
+import { connectMongoDB } from '@/lib/mongodb';
+import User from '@/schemas/user';
+import { compare } from 'bcrypt';
+
+const handler = NextAuth({
+  callbacks: {
+    async signIn({ user, account }) {
+      return true;
+    },
+    async jwt({ token, account, profile }) {
+      token = {
+        ...token,
+      };
+
+      return token;
+    },
+  },
+  providers: [
+    CredentialsProvider({
+      id: 'credentials',
+      name: 'Credentials',
+      credentials: {
+        password: {
+          label: 'Password',
+          type: 'password',
+        },
+        email: {
+          label: 'Email',
+          type: 'text',
+        },
+      },
+
+      async authorize(credentials) {
+        await connectMongoDB();
+
+        const user =
+          credentials?.email !== ' '
```

Comment:
Why are we checking if it's a single space string?

Reply:
vestiges of another project

Reply:
?